skyportal/tests/frontend/test_reminders.py

```diff
@@ -0,0 +1,192 @@
+import os
+import uuid
+import time
+
+from skyportal.tests import api
+from datetime import date, timedelta, datetime
+from selenium.webdriver.common.keys import Keys
+
+
+def post_and_verify_reminder(endpoint, token):
```

Comment:
Try to import this from the backend test.

Reply:
I'm having trouble importing it. This function used the API method, and once it is imported it can't access the API method and it gives me an error:
```
TypeError: 'module' object is not callable
```

Reply:
I'm guessing you import `skyportal/tests/api` as a module, then that name overrides the function `api`. Try doing something like `from skyportal/tests/api import post_and_verify_reminder`.